pkg/resourcemonitor/types.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcemonitor
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+
+	topologyv1alpha1 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha1"
+)
+
+// Args stores commandline arguments used for resource monitoring
+type Args struct {
+	PodResourceSocketPath string
+	SleepInterval         time.Duration
+	Namespace             string
+	FsRoot                string
```

Comment:
Related to the cmdline flag, I think you could drop this, too

Reply:
Sure, will remove